sdk/src/test/java/io/opentelemetry/sdk/trace/samplers/ProbabilitySamplerTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace.samplers;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Unit tests for {@link ProbabilitySampler}. */
+@RunWith(JUnit4.class)
+public class ProbabilitySamplerTest {
```

Comment:
It would be nice if we could have some tests asserting the behavior of `shouldSample`.  Maybe a nice opportunity to add some data driven testing?

Reply:
Good suggestion. I didn't copy all the tests just now, added them in https://github.com/open-telemetry/opentelemetry-java/commit/2e3eba23ea75b341ce053aebef940f9404683d3f.